projects/uitdatabank/docs/terms.md

```diff
@@ -1,7 +1,95 @@
 # Terms
```

Comment:
Maybe we should also mention somewhere (prominent) that you need to use the term ids from our Taxonomy Terms API, and you cannot use your own term ids / labels?

Reply:
Added this in a new commit

Reply:
Clear! 👍